sonar-php-plugin/src/test/java/org/sonar/plugins/php/phpunit/PhpUnitResultParserTest.java

```diff
@@ -117,6 +117,16 @@ public void shouldGenerateTestsMeasures() {
     verify(context).saveMeasure(bananaFile, CoreMetrics.TEST_EXECUTION_TIME, 570.0);
   }
 
+  /**
+   * Should not fail if system output is found
+   */
+  @Test
+  public void shouldNotFailIfSystemOutputIsFound() {
+    parser.parse(TestUtils.getResource(MockUtils.PHPUNIT_REPORT_DIR + "phpunit.systemout.xml"));
+
+    verify(context, never()).saveMeasure(any(org.sonar.api.resources.File.class), any(Metric.class), anyDouble());
```

Comment:
My mistake @vilchik-elena . Tried to find a simple unit test to use as basis, but without much knowledge about the code base I probably overcomplicated it.